### DISCUSSION TITLE
重启mivlus后，其中一个collection无法查询，一段时间后提示超时

### DISCUSSION THREAD

**Discussion by liuyichen2013** [milvus-io/milvus#16455] [Q&A and General discussion]
重启mivlus后，其中一个collection无法查询，一段时间后提示超时，其它collection可以正常查询，麻烦解惑，谢谢！
![2](https://user-images.githubusercontent.com/88218731/162710130-a801159b-68b7-41fa-956b-d17b5106a4a5.jpg)

**Comment by yhmo** [comment]
这种问题一言难尽，最好开一个issue，把搜集的日志贴上。
